examples/react-datastore/src/datasync/schema.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
@Eunovo think this file can go now. We have the `schema` subdirectory with the `schema.json` and `index.ts`

Reply:
Nevermind, you've done that already